lib/trento/domain/sap_system/sap_system.ex

```diff
@@ -197,7 +198,13 @@ defmodule Trento.Domain.SapSystem do
     sap_system
     |> Multi.new()
     |> Multi.execute(fn sap_system ->
-      emit_application_instance_registered_or_application_instance_health_changed(
+      maybe_emit_application_instance_registered_or_moved_event(
```

Comment:
Nice!
This multi opens up an easy way to add some `ApplicationInstanceUpdated` kind of event if we need it in the future